src/pkg/projects/dir.props

```diff
@@ -95,6 +95,11 @@
           Include="$(PackageThirdPartyNoticesFile)" >
       <SkipPackageFileCheck>true</SkipPackageFileCheck>
     </File>
+    <!-- Add a marker to help the designer optimize & share .NET Core packages -->
+    <File Condition="'$(IncludeDesignerMarker)' != 'false'"
+          Include="$(MSBuildThisFileDirectory)useSharedDesignerContext.txt">
```

Comment:
Be careful about this.  @	lutzroeder is this something you expect to see in the shared framework packages?  I suspect the designer might see these when designing for a self-contained app (since it runs against the package and not the shared framework).

Reply:
Where does this file get deployed exactly, who reads it, and how do they locate it?

Fixing tag, @lutzroeder, cc @chabiss

Reply:
Judging by https://github.com/dotnet/corefx/commit/8d7870ad127d5b25968835ceb22a6c87eb7e73f8 message, I don't think this should be needed for .NET Core 3.

Reply:
//sidebar

That's the only place I found any "documentation" ;-) about _useSharedDesignerContext.txt_ when I was moving `Microsoft.NET.Sdk.WindowsDesktop` to [dotnet/wpf](https://github.com/dotnet/wpf) (thanks @ericstj!). 

Is it documented officially anywhere else - or who should perhaps own the documentation for that file? Would a developr ever need or want to incorporate that into their own package? 

/cc @chabiss

Reply:
I'll get rid of this unless I get confirmation otherwise, only motivation was package content parity. Copying what I mentioned in the PR description here for context:

> This PR adds the empty `useSharedDesignerContext.txt` file to all Core-Setup packages. It looked like Core-Setup was simply missing this. (Seems obsolete for these packages, but figured might as well to remove the difference.)